src/views/AuthorityView/AuthorityView.js

```diff
@@ -40,6 +40,7 @@ import useAuthorityDelete from '../../queries/useAuthoritiesDelete/useAuthorityD
 
 const propTypes = {
   authority: PropTypes.shape({
+    allData: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
I can't find - where does this property come from?

Reply:
Forgot to assign you to [pull request](https://github.com/folio-org/stripes-authority-components/pull/50)